onnxruntime/core/common/profiler.h

```diff
@@ -60,9 +60,11 @@ class Profiler {
     return enabled_;
   }
   /*
-  Return the stored start time of profiler
+  Return the stored start time of profiler.
+  On Windows and Mac, this timer is not as precise as nanoseconds. 
+  The precision of high_resolution_clock is ~100ns.
```

Comment:
Would be good to add these comments in the C API header as well for SessionGetProfilingStartTimeNs. Can be done in a separate PR.